### PR TITLE
Scale video to frame and original size. you can resize the project view and dokument list by draging 

### PR DESCRIPTION
**Pull request by svidem**

### CODE REVIEW

**Comment by parso619** [comment]
Why is there a menu option "Fill screen"? The video automatically rescales when the QLabel is changed so it will always fill the screen (when not in "Original size"-mode)? I can't find any scenario where the "Fill screen" option has any effect.

File: ViAn/GUI/mainwindow.h [hunk 1]
```diff
@@ -23,6 +23,7 @@
 #include "qtreeitems.h"
 #include <QMutex>
 #include <QWaitCondition>
+#define SCROLLAREAMARGIN 25
```
Comment (henpe071) [inline ViAn/GUI/mainwindow.h]: I would like some underscores in the constant

File: ViAn/GUI/mainwindow.ui [hunk 2]
```diff
@@ -17,35 +17,44 @@
    <double>1.000000000000000</double>
   </property>
   <widget class="QWidget" name="centralWidget">
-   <layout class="QVBoxLayout" name="verticalLayout_2">
+   <layout class="QVBoxLayout" name="verticalLayout">
```
Comment (henpe071) [inline ViAn/GUI/mainwindow.ui]: I know this is a overall problem in the ui file but these layouts really should be more descriptive. Something along the lines of projectTreeLayout would be better.

File: ViAn/Video/video_player.h [hunk 3]
```diff
@@ -107,11 +110,15 @@ public slots:
...
 
+    // The QLabel where the video is shown.
+    QLabel* video_frame;
+
     cv::VideoCapture capture;
     cv::Mat frame;
     cv::Mat RGBframe;
 
     int num_frames;
+
```
Comment (henpe071) [inline ViAn/Video/video_player.h]: this line isn't necessary

Comment (parso619) [inline ViAn/Video/video_player.h]: fixed

File: ViAn/GUI/mainwindow.h (hunk 1 above)
Comment (parso619) [inline ViAn/GUI/mainwindow.h]: fixed

File: ViAn/GUI/mainwindow.ui (hunk 2 above)
Comment (parso619) [inline ViAn/GUI/mainwindow.ui]: This layout is for the whole centralwidget (including the project view, video player, and bookmark view) and within this layout there is a splitter with horisontal orientation. So your proposed name would be incorrect here.

Comment (svidem) [inline ViAn/GUI/mainwindow.ui]: Becouse they are not used for anything outside the .ui file there is no need for a descriptive name

**Pull request merged**